discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -52,12 +53,17 @@ def generate_and_download_image(
             image_path=iso_download_path
         )
 
-    def wait_until_hosts_are_discovered(self,nodes_count=env_variables['num_nodes']):
+    def wait_until_hosts_are_discovered(self, nodes_count=env_variables['num_nodes'],
```

Comment:
@avishayt why we need additional_statuses ? why i can't do it with  wait_till_all_hosts_are_in_status?